src/app/tables/table.component.ts

```diff
@@ -39,6 +39,8 @@ export class EstuaryTableComponent implements OnChanges {
   @Input() preformattedColumns: Array<string>;
   // Optional margin spacing surrounding the table
   @Input() tableSpacing = true;
+  // Link to an artifact's corresponding story
+  @Input() nodeStoryUrls: any;
```

Comment:
@mprahl The reason I wasn't using linkColumnMappings is because I wanted to distinguish it as its own thing separate from normal links. It does not get the same treatment as a normal column (ie no column header, not included on the CSVs) so I wanted to put it in an entirely separate thing to highlight that it is different from the other columns. If that makes sense? Curious to hear your thoughts. I could still do a similar thing but just change the variable names maybe?